integration-libs/s4-service/root/model/checkout-service-details.model.ts

```diff
@@ -14,3 +14,13 @@ export interface ServiceOrderConfig {
   leadDays?: number;
   serviceScheduleTimes?: string[];
 }
+
+export interface CancelObj {
```

Comment:
`minor:` I'm not fond to have interface name that contains `object`, `obj` and similar in it's name as well as incomplete words. Apart from that, it doesn't convey the purpose of the interface clearly. Let's consider names that are descriptive - what do you think about:
- `CancellationRequest`
- `CancellationDetails`